src/tasks/DarkTask.py

```
@@ -0,0 +1,87 @@
+            {
+                self.CONF_TIME: 0,
+            }
+        )
+
+    def run(self):
+        super().run()
+        try:
+            self.do_run()
+        except TaskDisabledException:
+            pass
+        except Exception as e:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)